src/explore-education-statistics-admin/src/pages/publication/components/PublicationAddExistingUsersTab.tsx

```diff
@@ -0,0 +1,104 @@
+import WarningMessage from '@common/components/WarningMessage';
+import React from 'react';
+import releasePermissionService, {
+  ManageAccessPageContributor,
+} from '@admin/services/releasePermissionService';
+import { Formik } from 'formik';
+import { Form, FormFieldCheckboxGroup } from '@common/components/form';
+import Button from '@common/components/Button';
+import useFormSubmit from '@common/hooks/useFormSubmit';
+import { generatePath, useHistory } from 'react-router-dom';
+import { publicationManageTeamAccessReleaseRoute } from '@admin/routes/routes';
+import { ReleaseRouteParams } from '@admin/routes/releaseRoutes';
+import { Release } from '@admin/services/releaseService';
+
+interface AddExistingUsersFormValues {
+  users: string[];
```

Comment:
Just wondering: Are `users` an actual string array in this instance (i.e. bob@bob.com, test@test.com), or could we be more specific by specifying a stricter type?

Reply:
It's user ids. I'll rename the variable to `userIds` be clearer.